modules/mesh-task/variables.tf

```diff
@@ -275,64 +281,32 @@ variable "consul_datacenter" {
 
 variable "application_shutdown_delay_seconds" {
   type        = number
   description = <<-EOT
-  Set an application entrypoint to delay the TERM signal from ECS for this many seconds.
-  This allows time for incoming traffic to drain off before your application container exits.
-  This cannot delay the KILL signal from ECS, so this delay should be shorter than the `stopTimeout`
-  on the container definition.
-
-  This will set the `entryPoint` field for each container in `container_definitions` that does not have
-  an `entryPoint` field. Containers with a non-null `entryPoint` field will be ignored. Since this sets
-  an explicit entrypoint, the default entrypoint from the image (if present) will not be used. You may
-  need to set the `command` field on the container definition to ensure the container starts properly.
+  An optional number of seconds by which to delay application shutdown. By default, there is no delay. This delay allows
+  incoming traffic to drain off before your application container exits. This delays the TERM signal from ECS when
+  the task is stopped. However, the KILL signal from ECS cannot be delayed, so this value should be shorter than the
+  `stopTimeout` on the container definition. This works by setting an explicit `entryPoint` field on each container without an
+  `entryPoint` field. Containers with a non-null `entryPoint` field will be ignored. Since this sets an explicit entrypoint,
+  the default entrypoint from the image (if present) will not be used, so you may need to set the `command` field on the
+  container definition to ensure your container starts properly, depending on your image.
   EOT
-  default     = null
+  default     = 0
 }
 
 variable "consul_ecs_config" {
   type        = any
   default     = {}
-  description = <<EOT
+  description = <<-EOT
   Additional configuration to pass to the consul-ecs binary for Consul service and sidecar proxy registration requests.
-
-  This accepts a subset of the [consul-ecs config file](https://github.com/hashicorp/consul-ecs/blob/main/config/schema.json).
+  This accepts a subset of the consul-ecs config file (https://github.com/hashicorp/consul-ecs/blob/main/config/schema.json).
   For the remainder of the consul-ecs config file contents, use the variables `upstreams`, `checks`, `consul_service_name`,
-  `consul_service_tags`, `consul_service_meta`, `consul_namespace`, and `consul_partition`. In most cases, these separate variables will suffice.
-
-  Example:
-  ```
-  consul_ecs_config = {
-    service = {
-      enableTagOverride = false
-      weights = {
-        passing = 1
-        warning = 1
-      }
-    }
-    proxy = {
-      config = {}
-      meshGateway = {
-        mode = "remote"
-      }
-      expose = {
-        checks = true
-        paths = [
-          {
-            listenerPort = 1234
-            path = "/path"
-            localPathPort = 2345
-            protocol = "http"
-          }
-        ]
-      }
-    }
-  }
-  ```
```

Comment:
This condenses to a single line in the TF registry, so I opted to just remove the example. These are infrequently used options anyway so I doubt it's that helpful.